src/librustc_middle/ty/context.rs

```diff
@@ -314,6 +314,8 @@ pub struct GeneratorInteriorTypeCause<'tcx> {
     pub span: Span,
     /// Span of the scope of the captured binding.
     pub scope_span: Option<Span>,
+    /// Span of `.await` statement.
+    pub await_span: Span,
```

Comment:
Since this is more generally for generators, can you call it `yield_span` and comment with "Span of `yield` or `.await` expression"?